src/components/NewLocationPage/LocationOverview/LocationOverview.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import OverallRiskBlock from '../OverallRiskBlock';
+import SummaryStat from '../SummaryStat';
+import VaccinationProgressBarBlock from '../VaccinationProgressBarBlock';
+import {
+  GridContainer,
+  GridItemLevel,
+  GridItemProgress,
+  GridItemMetricVax,
+  GridItemMetric1,
+  GridItemMetric2,
+  GridItemMetric3,
+} from './LocationOverview.style';
+import { SectionContainer } from '../Shared/Shared.style';
+import { Region } from 'common/regions';
+import { LocationSummary } from 'common/location_summaries';
+import { summaryToStats } from '../SummaryStat/utils';
+import { Metric } from 'common/metricEnum';
+
+const LocationOverview: React.FC<{
+  region: Region;
+  locationSummary: LocationSummary;
+}> = ({ region, locationSummary }) => {
+  const stats = summaryToStats(locationSummary);
+
+  return (
+    <SectionContainer style={{ border: '1px dotted gray', maxWidth: '880px' }}>
```

Comment:
for QA: 880px is the true max width of the section containers in the mocks. also dotted border for qa